plugins/web-worker-offloading/helper.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Helpers for Web Worker Offloading.
+ *
+ * @since 0.1.0
+ * @package web-worker-offloading
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Gets configuration for Web Worker Offloading.
+ *
+ * @since 0.1.0
+ * @link https://partytown.builder.io/configuration
+ * @link https://github.com/BuilderIO/partytown/blob/b292a14047a0c12ca05ba97df1833935d42fdb66/src/lib/types.ts#L393-L548
+ *
+ * @return array<string, mixed> Configuration for Partytown.
+ */
+function wwo_get_configuration(): array {
+	$config = array(
+		'lib' => wp_parse_url( plugin_dir_url( __FILE__ ), PHP_URL_PATH ) . 'build/',
+	);
+
+	if ( WP_DEBUG && SCRIPT_DEBUG ) {
+		$config['debug'] = true;
+	}
```

Comment:
How about always having this key set, and making the conditional only around whether it's `true` or `false`?

Reply:
Well, because it defaults to `false` anyway, and by not setting it we avoid sending (a tiny amount) of bytes down the wire.

Reply:
That's probably negligible though for performance. I'd say initializing properties here makes it easier to understand what's available, also since this PHP layer is entirely separate from the actual JS API where this is consumed.

Reply:
I'm not sure. If someone wants to know what is available they should look at the docs which are linked in the source code:

```
	 * @link https://partytown.builder.io/configuration
	 * @link https://github.com/BuilderIO/partytown/blob/b292a14047a0c12ca05ba97df1833935d42fdb66/src/lib/types.ts#L393-L548
```

Reply:
We don't want to have to supply the default values for very single configuration option (many of which are under-documented in Partytown).

Reply:
That's fair. Let's leave this as is then.